### DISCUSSION TITLE
Problem with jQuery link after copying site to different machine

### DISCUSSION THREAD

**Discussion by NJoco** [OrchardCMS/OrchardCore#9390] [Q&A]
Hi guys,

I ran into a problem when I tried to move a working Orchard Core site from one place to another.
I simply copied everything and created site on IIS with the same settings.
The page loaded but the theme and many other things are not working.
I found the following error several times in the browsers developer console:
Error parsing 'integrity' attribute ('sha384-vk5WoKIaW/vJyUAd9n/wmopsmNhiy<!DOCTYPE html>
<html lang='). The digest must be a valid, base64-encoded value.

Error parsing 'integrity' attribute ('sha384-vk5WoKIaW/vJyUAd9n/wmopsmNhiy<!DOCTYPE html>
<html lang='). The hash algorithm must be one of 'sha256', 'sha384', or 'sha512', followed by a '-' character

Uncaught SyntaxError: Unexpected token '<'

After some digging in the code I found that the link of jQuery is generated in a strange (and invalid) way:
`<script crossorigin="anonymous" integrity="sha384-vk5WoKIaW/vJyUAd9n/wmopsmNhiy<!DOCTYPE html>`
`<html lang=" en-us"="">`
`<head>`
`" src="https://code.jquery.com/jquery-3.4.1.min.js" type="text/javascript">`

Seems like a piece of HTML would be inserted between two attributes. 

Do you have any idea what can be the problem and how can I solve it?

Thank you in advance for any help.

**Comment by Skrypt** [comment]
Looks like an issue that has been fixed in latest dev branch. This is simply bad HTML rendering because of a patch that has been applied in .NET Core. Try publishing as "Self contained" to basically not use the version installed on the server if you want to avoid the upgrade to latest version of OC. Else, I recommend doing so.
